nncf/onnx/graph/node_utils.py

```diff
@@ -93,3 +88,34 @@ def get_input_edge(input_node_name: str, input_edges_mapping: Dict[str, Tuple[st
         input_edges.add(onnx_graph.get_node_edge_names(name)["input"][port_id])
     assert len(input_edges) == 1
     return input_edges.pop()
+
+
+def is_any_weight_quantized(node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
```

Comment:
IMHO, it's better to call this method `is_weights_quantized` since `any` adds some kind of uncertainty.

Reply:
`any` adds understanding that if there are different weights it is enough to only one of them to be quantized to return True.

Reply:
Not agree.